Test.cs

```diff
@@ -12,7 +14,12 @@ public static long calculate(long number)
                 throw new Exception("Number must be non-negative.");
             }
```

Comment:
The comment should be updated to reflect the correct functionality of the method, mentioning that it calculates the factorial of a non-negative integer. Additionally, ensure that all comments are up-to-date and accurately describe the method or field they are associated with.